product_docs/docs/efm/4/04_configuring_efm/05_using_vip_addresses.mdx

```diff
@@ -10,13 +10,39 @@ legacyRedirectsGenerated:
 
 <div id="using_vip_addresses" class="registered_link"></div>
 
-Failover Manager uses the `efm_address` script to assign or release a virtual IP address.
+Failover Manager can be used along with a virtual IP address (VIP) for routing requests to the current primary node. 
 
 !!! Note "Cloud provider support and alternatives"
     Virtual IP addresses aren't supported by many cloud providers. In those environments, use another mechanism, such as an elastic IP address on AWS, that can be changed when needed by a fencing or post-promotion script.
 
-!!! Note "Behavior when the agent is stopped"
-    Failover Manager will not drop the Virtual IP address (if used) from the primary node when the agent is stopped. As a convenience for testing, the primary agent will acquire the VIP at startup if the node does not already have it, but otherwise starting and stopping Failover Manager has no effect on the node having the virtual IP address.
+## Behavior when the EFM agent is stopped
```

Comment:
Am concerned someone might think "is stopped" means "WHILE the agent is stopped" as opposed to the action of stopping it. It's obvious to you/me that efm doesn't do anything with the VIP while it's not running, but you never know how someone else might interpret it. Could it be more like "Behavior when the [primary] agent exits"? Am fine with/without "primary" in the text since it's clear from the description below.

Reply:
This is a great observation - I think let's be explicit about which process we're describing here. I'm gonna suggest "during shutdown" - this also aligns the wording with the section below.

I've also tweaked the paragraph that follows to further emphasize the distinction between the lifecycle of a *node* and the *agent* for that node.